cyclictest-client

```diff
@@ -88,8 +88,7 @@ echo -e "${CMD_OUTPUT}"
 HK_CPUS=$(echo -e "${CMD_OUTPUT}" | grep "final cpus:" | awk '{ print $3 }')
 echo "HK_CPUS: ${HK_CPUS}"
 
-
-cmd="taskset -c $WORKLOAD_CPUS,$HK_CPUS /usr/bin/cyclictest --policy $policy -i $interval -H 1000 --histfile histogram.txt -D $duration -q -a ${WORKLOAD_CPUS} -t"
+cmd="taskset -c $HK_CPUS /usr/bin/cyclictest --policy $policy -i $interval -H 1000 --histfile histogram.txt -D $duration -q -a ${WORKLOAD_CPUS} -t"
```

Comment:
is the workload_cpus in taskset not necessary because cyclictest is just going to override it anyway for the "worker" threads?

Reply:
That is my understanding.  Further, if CPU load balancing is enabled it is possible that the "main" thread would be migrated onto one of the worker thread CPUs.